build.gradle

```diff
@@ -0,0 +1,102 @@
+import net.fabricmc.loom.task.RunGameTask
```

Comment:
out of curiosity, why put this configuration here instead of in a buildSrc convention plugin?

Reply:
Mainly laziness since loom was already on the classpath from the settings (and isn't in buildSrc), also I think it was like this in your draft branch that I started with.

Since we use `repositoriesMode.set(RepositoriesMode.PREFER_SETTINGS)` and therefore need to apply loom to the settings regardless, we would need to declare the loom version in the version catalog and settings to move it. (Unless we use some complicated work around like https://discuss.gradle.org/t/how-to-use-version-catalog-in-the-root-settings-gradle-kts-file/44603/5)

Reply:
Actually, since we don't use loom for NeoForge anymore, it may be simpler to resolve. Since we don't care about runs in the repack project.